src/Infrastructure/LeanCode.Firebase.FCM/PushNotificationTokenStore.cs

```diff
@@ -68,9 +68,11 @@ public async Task AddUserTokenAsync(TUserId userId, string newToken, Cancellatio
             var dateCreatedColumn = GetTokensColumnName(nameof(PushNotificationTokenEntity<TUserId>.DateCreated));
 
             // perform an upsert using SQL merge, ensuring that given token is assigned to our user and no-one else
-            await dbContext.Database.ExecuteSqlInterpolatedAsync(
-                FormattableStringFactory.Create(
-                    $$"""
+            await dbContext
+                .Database
+                .ExecuteSqlInterpolatedAsync(
+                    FormattableStringFactory.Create(
+                        $$"""
```

Comment:
That seems tragic 😞. Not aligned anymore.

Reply:
+1, does CSharpier enforce this exact formatting or can we indent the content and closing `"""` a bit without invoking its wrath?

Reply:
Yep, indent calmed it down